groups.py

```diff
@@ -978,7 +993,7 @@ def group_create(ctx, group_name, category, subcategory, schema_id, expiration_d
         message = response[9]
         if status == '0':
             return api.Result.ok()
-        elif status == '-1089000' or status == '-809000':
+        elif status == '-1089000' or status == '-809000' or status == '-806000':
```

Comment:
Good idea to add the check for SQL errors here.

On production environments we'd like to avoid SQL errors as much as possible, since they result in a significant amount of hard-to-read error messages in the logs. Can you also add a check to see if the group already exists just above the uuGroupAdd invocation? In that case, we should only get these SQL errors when two people are trying to create the same group simultanteously, which should be very rare.

Reply:
Done! Added check